index.js

```diff
@@ -83,7 +117,15 @@ function countByRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+  let foundMovie = null;
+  for (let i = 0; i < movies.length; i++) {
+    if (movies[i]["imdbId"].includes(id)) {
```

Comment:
Aww man we were so close here 😔. The key in the movie object is actually `imdbID` not `imdbId`.